components/WhatICanHelpYouWith/organisms/Bottom.tsx

```diff
@@ -0,0 +1,41 @@
+import { IndexPageV2DocumentDataServiceGroupItem } from 'prismicio-types';
+import { KeyTextField } from '@prismicio/client';
+
+import { LowerContainer, MobileCTAContainer } from 'components/WhatICanHelpYouWith/organisms/WhatICanHelpYouWith.styles';
+import { items } from 'components/WhatICanHelpYouWith/content/items';
+import Unit from 'components/WhatICanHelpYouWith/molecules/Unit';
+import PrimaryCTAButton from 'components/atoms/PrimaryCTAButton';
+import PsychoLink from 'components/atoms/PsychoLink';
+import { Grey010, Green100 } from 'components/styling/colors';
+
+interface BottomProps {
+  serviceGroups: IndexPageV2DocumentDataServiceGroupItem[];
+  priceListButtonText: KeyTextField;
+}
+
+const Bottom = ({ serviceGroups, priceListButtonText }: BottomProps) => (
+  <LowerContainer>
+    {items.map(
+      (item, index) => (
+        <Unit
+          key={index}
+          item={item}
+          title={serviceGroups[index].service_name ?? ''}
```

Comment:
This can be broken when the order is modified. Hard to debug in this way, rather use IDs if we can

Reply:
You are using indexes and that is order specific. If we change order in CMS for any reason this will result in weird bugs

Reply:
Do you mean the icons not being correct for the given service? Otherwise it will go through and render as much service as it finds on the prismic repo.

Reply:
Yeah I mean exactly that. That's why I say that it would be more beneficial to use a `Map()` here. I am pretty sure the CMS item should have some kind of unique identifier to it.